### ISSUE TITLE
Relative paths starting with '../' are parsed incorrectly in join()

### ISSUE THREAD

**Issue by vuorinem** [aurelia/path#27]
**I'm submitting a bug report**

* **Library Version:**
1.1.1

**Current behavior:**
```
join('../other/app','/url')
// result: 'other/app/url'
```

**Expected/desired behavior:**
```
join('../other/app','/url')
// result: '../other/app/url'
```

When using parent path reference, `join()` removes the previous value from the path array. If the parent reference is at the start of the path, it gets removed and ignored:

https://github.com/aurelia/path/blob/1.1.1/src/index.js#L87-L88

This could be fixed by leaving the parent reference in the path if it is at the start of the path.

**Comment by davismj** [comment]
As you noted, a very simple fix. I'm not totally sure the implications of this change so plenty of good testing is needed. PRs welcome.

**Comment by 3cp** [comment]
This will be fixed by #35.
